### PR TITLE
Chapter 4 errata

### PR DESCRIPTION
**Pull request by polaris-**
I finished going over chapter 4 today and here's what I found. I also found some other stuff that I wasn't entirely sure about, so I'll make a note of it here and let someone else decide what to do:
- Softmax prediction section:
  Because we're reusing read_csv() from the logistic regression section, read_csv() still has skip_header_lines=1 even though there is no header in the iris data. This results in 96% instead of the stated 95% in the book during my runs. Removing that option results in 95% as stated.
- Page 120:
  "This problem actually made neural network research to loss importance for about a decade around 1970’s."
  I feel like there are a few problems in this line but I might just not be understanding it.
- Page 121: 
  "You can think of it as allowing our network to ask multiple questions to the input data, one question per neuron on the hidden layer. And finally decide the output result based on the answers of those questions."
  Is the "And" sentence starting its own sentence intentional?
- Page 130:
  ∂loss / ∂w<sub>2</sub> = loss′ . L3′ . L2′ . L1′ . x
  Should this really be ∂w<sub>2</sub> here? I'm not confident enough in my math to say for sure.

### CODE REVIEW

**Comment by samjabrahams** [comment]
Thanks so much! In response to the other things:

-Softmax prediction: We may not change the percentage amount due to randomly initializing the weights, which may lead to slightly different results, though we'll probably tweak the code to explicitly turn off the `skip_header_lines` option just so that we're being precise in our strokes
- Looks like a grammar hiccup that made it through editing
- Ditto
- Good catch! That should be Thanks so much! In response to the other things:

-Softmax prediction: We may not change the percentage amount due to randomly initializing the weights, which may lead to slightly different results, though we'll probably tweak the code to explicitly turn off the `skip_header_lines` option just so that we're being precise in our strokes
- Looks like a grammar hiccup that made it through editing
- Ditto
- Good catch! That should be ∂loss/∂w1.

We're going through the errata and making fixes to the book, and you should be able to download a newer version very soon. Thanks for your help!

**Comment by polaris-** [comment]
Thanks, looking forward to downloading the newer version whenever you guys get it prepared!

I'm currently half way through chapter 5 now and so far I haven't really found any real issues besides hyperlink-related stuff. I might submit an issue for those later since they're not so much errors in the print.
